### PR TITLE
Content sync and updated to md | Element.animate [ES]

### LINKED ISSUE #4077: 12. /es/docs/web/api/element/animate - [en-US]

### PR DESCRIPTION
**Pull request by Vallejoanderson**
Fixes #4077

### CODE REVIEW

File: files/es/web/api/element/animate/index.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+];
+```
+
+Solamente hemos especificado el fin del estado de la animación, y el estado inicial está implícito.
+
+## Especificaciones
+
+{{Specifications}}
+
+## Compatibilidad con Navegadores
```
Comment (JuanVqz) [inline files/es/web/api/element/animate/index.md:75]:
```suggestion
## Compatibilidad con navegadores
```

**Review by JuanVqz — COMMENTED** [review]
:rocket:

File: files/es/web/api/element/animate/index.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+- `options`
+  
+  - : Puede ser un **entero representando la duración de la animación** (en milisegundos), **o** un 
+    objeto que contiene una o más propiedades acerca del tiempo descritas en los 
+    [los parametros de opciones del `KeyframeEffect()`](/es/docs/Web/API/KeyframeEffect) y/o las 
+    siguientes opciones:
+   
+    - `id {{optional_inline}}`
+      - : Una propiedad única `animate()` para animar: una [`DOMString`](/es/docs/Web/API/DOMString) 
+      la cual hace referencia a la animación.
```
[the remark this answers is not in the document]
Comment (Vallejoanderson) [reply files/es/web/api/element/animate/index.md]: Copiado, gracias ;)

**Review by davbrito — APPROVED** [review]

**Review by JuanVqz — APPROVED** [review]
Excelente!

**Pull request merged**